src/wazuh_db/wdb.h

```diff
@@ -875,6 +923,17 @@ wdb_chunks_status_t wdb_sync_agent_info_get(wdb_t *wdb, int* last_agent_id, char
  */
 int wdb_global_sync_agent_info_set(wdb_t *wdb, cJSON *agent_info);
 
+/**
+ * Doxygen here
```

Comment:
Doxygen description style will be added.

Reply:
Fixed